traitsui/testing/tester/wx/default_registry.py

```diff
@@ -14,6 +14,8 @@
 from traitsui.testing.tester.wx.implementation import (
     button_editor,
     check_list_editor,
+    editor_factory_readonly_editor,
+    editor_factory_text_editor,
```

Comment:
Nit: I think I'd prefer the module matching the that in `traitsui.qt4` so that it is easier to find them. This is a bit like the convention of naming test files according to the module being tested. But then there are always exceptions to the rules. Is there a reason for these two to be separated?

Reply:
> Is there a reason for these two to be separated?

Not a good one, I will make the change to mirror that of `traitsui.qt4` as that makes more sense.